index.html

```diff
@@ -26,9 +28,26 @@
                  <img src="images/image_geometry_1.png" alt="image">
              </div>
              <nav id="header-nav-lg">
-                 <a href="#" id="nav-expand-lg"><i class="fa fa-bars"></i></a>
+                 <a id="nav-expand-lg" class="open-menu" data-open="menu2"><i class="fa fa-bars"></i></a>
              </nav>
             </div>
+            <div class="menu" id="menu2">
+                <div class="menu-dialog">
+                  <h3 class="menu-header">
+                    <a class="close-menu" aria-label="close menu" data-close>
+                      ✕  
+                    </a>
+                  </h3>
+                  <div class="mobile-nav">
+                    <ul data-close>
+                     <li><a href="#hello">Hello</a></li>
+                     <li><a href="#projects">Portfolio</a></li>
+                     <li><a href="#about">About</a></li>
+                     <li><a href="#contact">Contact</a></li>
+                    </ul>
+                </div>
+                </div>
+              </div>
```

Comment:
- [ ] You have done a great job in implementing this mobile menu, but the height seems not to be stable at different screen sizes. Please make sure the height is the full screen height when on mobile view for any view width. You can use `top:0` and `bottom:0` instead of `height` and `max-height`

### Your Design
![image](https://user-images.githubusercontent.com/43995830/158006286-eb4a64d1-fcb7-4cff-9e1e-27472aab20d1.png)

Original Design
![image](https://user-images.githubusercontent.com/43995830/158006298-ef28a996-2856-4852-9d69-40a9ee3a2de4.png)